tests/utils.py

```diff
@@ -219,16 +219,13 @@ def mscolab_start_server(all_ports, mscolab_settings=mscolab_settings, timeout=1
         args=(_app, sockio, cm, fm,),
         kwargs={'port': port})
     process.start()
```

Comment:
I disagree, that try-except would make it worse. This process is started in a pytest fixture, if an exception occurs there then this will be reported as an test error for thst test, with the same output that pytest does for exceptions usually. I.e. similar logging to what was suggested. If we catched it and just logged, then we would run into the loop afterwards and introduce the additional delay.

I am also not sure which exceptions are even possible to come from Process and .start, this would catch none of the possible exceptions in the application code.

Reply:
I just realized your comment was from 3 weeks ago and it was not yet a fixture then. It is now though.